spec/granite/query/assemblers/sqlite_spec.cr

```diff
@@ -9,18 +9,18 @@ require "../spec_helper"
       end
 
       it "simple counts" do
-        builder.count.raw_sql.should match ignore_whitespace "select count(*) from table"
+				sql = "select count(*) from table"
+        builder.count.raw_sql.should match ignore_whitespace sql
       end
-    end
 
-	context "counts" do
-      it "adds group_by fields for where/count queries" do
+			it "adds group_by fields for where/count queries" do
         sql = "select count(*) from table where name = ? group by name"
-        builder.where(name: "bob").counts.raw_sql.should match ignore_whitespace sql
+        builder.where(name: "bob").count(true).raw_sql.should match ignore_whitespace sql
```

Comment:
passing a boolean to `count` is not intuitive and is a breaking change for existing users.  To be honest, I don't see why we need a `group by name`  added at all here.  Only if someone specifies a `group_by` should it be included.